src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
 {
+    public Binding_Http_BasicHttpBindingTests() : base("WcfService.TestResources.BasicHttpResource") { }
+
     [Fact]
     [OuterLoop]
-    public static void DefaultSettings_Echo_RoundTrips_String()
+    public void DefaultSettings_Echo_RoundTrips_String()
```

Comment:
I presume the elimination of static here also reflects all the OuterLoopTests are instance-based now.  Is that strictly necessary? What are the pro's and con's of doing this versus interacting with a singleton Bridge instance?